src/Pages/TransitPredictions/TransitPredictionsPage.js

```diff
@@ -0,0 +1,125 @@
+          <p className="font-light text-gray-500 sm:text-xl dark:text-gray-400">Transit information is retrieved from 511 SF Bay's Portal for Open Transit Data.</p>
+        </div>
+        <div className="flex flex-col items-center justify-center m-4 p-6 min-w-80 max-h-[50vh] text-xl
+         overflow-y-auto bg-white rounded-lg border border-gray-200 shadow-md dark:bg-gray-800 dark:border-gray-700">
+          <span className="text-center"><b>Transit predictions are retrived from SCEta, but it's disabled by default.</b></span>
+          <p>To enable SCEta:</p>
+          <ol className='list-decimal py-2'>
+            <li>Modify{' '}
+              <code>
+                api/config/config.json
+              </code>
+              {' '}to include{' '}
+              <code>
+                'SCEta': &#123;
+                'ENABLED': true
+                &#125;
+              </code>
+            </li>
+            <li>
+              Then, clone SCEta locally and follow the steps in{' '}
+              <a href="https://github.com/SCE-Development/SCEta/blob/master/README.md" target="_blank"
+                className="underline text-blue-600 hover:text-blue-800 visited:text-purple-600">
+                the readme
+              </a>
+              {' '} to run locally.
+            </li>
+            <li>
+              Run Clark and open this page again.
+            </li>
+          </ol>
+        </div>
+      </div>
+    );
+  }
+
+  if (error) {
+    return (
+      <div className="py-8 px-4 mx-auto lg:py-16 lg:px-6">
+        <div className="mx-10 text-center lg:mb-16 mb-8">
+          <h2 className="mb-4 text-3xl lg:text-4xl tracking-tight font-extrabold text-gray-900 dark:text-white">SCEta's Transit Predictions</h2>
+          <p className="font-light text-gray-500 sm:text-xl dark:text-gray-400">Transit information is retrieved from 511 SF Bay's Portal for Open Transit Data.</p>
+        </div>
+        <div className="flex flex-col m-4 p-6 min-w-80 max-h-[50vh] text-xl overflow-y-auto
+       bg-white rounded-lg border border-gray-200 shadow-md dark:bg-gray-800 dark:border-gray-700">
+          <span className="text-center"><b>Unable to load SCEta transit predictions</b></span>
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <section className="bg-white dark:bg-gray-900">
+      <div className="py-8 px-4 mx-auto lg:py-16 lg:px-6">
+        <div className="mx-10 text-center lg:mb-16 mb-8">
+          <h2 className="mb-4 text-3xl lg:text-4xl tracking-tight font-extrabold text-gray-900 dark:text-white">SCEta's Transit Predictions</h2>
+          <p className="font-light text-gray-500 sm:text-xl dark:text-gray-400">Transit information is retrieved from 511 SF Bay's Portal for Open Transit Data.</p>
+        </div>
+        <div className="grid gap-2 lg:grid-cols-2">
+          {busData.map((stop) => (
```

Comment:
paranoid because while `busData` *should* always be an array, the below just makes sure. if busdata was an object, `length` would be null

```suggestion
          {busData.length && busData.map((stop) => (
```

Reply:
fixed